File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/VMRecipient.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class VMRecipient
{
	[JsonProperty("user")]
	public UserDetails UserDetails { get; set; }

	[JsonProperty("is_phone")]
	public bool IsPhone { get; set; }

	[JsonProperty("is_email")]
	public bool IsEmail { get; set; }
}

File: Android/Vine-5.50.1/Decompilation/src/java/co/vine/android/scribe/model/VMRecipient.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;
import twitter4j.conf.PropertyConfiguration;

@JsonObject
/* loaded from: classes.dex */
public class VMRecipient {

    @JsonField(name = {"is_email"})
    public Boolean isEmail;

    @JsonField(name = {"is_phone"})
    public Boolean isPhone;

    @JsonField(name = {PropertyConfiguration.USER})
    public UserDetails user;
}
